core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -1348,6 +1348,7 @@ private[spark] object SparkSubmitUtils {
       coordinates: String,
       ivySettings: IvySettings,
       exclusions: Seq[String] = Nil,
+      transitive: Boolean = true,
```

Comment:
Nit: Add Scaladoc for this new parameter

Reply:
Done and `transitive` moved to front of `exclusions`

Reply:
What happens if we remove this default value? If it doesn't make many changes, let's remove this default value.

Reply:
> What happens if we remove this default value? If it doesn't make many changes, let's remove this default value.

we can remove this default value and then add this in place where use this method.
![image](https://user-images.githubusercontent.com/46485123/103049810-b871a580-45cd-11eb-8750-91eeab214ec4.png)

Reply:
> What happens if we remove this default value? If it doesn't make many changes, let's remove this default value.

Remove now and  you can see the change to judge where need to revert back.